main.py

```diff
@@ -13,12 +13,27 @@ def main():
     parser.add_argument('aab_file', type=str, help='Path of the Android App Bundle file')
     parser.add_argument('track', choices=['production', 'alpha', 'beta', 'internal'], default='internal',
                         help='Track to upload the apk to')
+    parser.add_argument(
+        '--release-notes', '-n',
+        type=str,
+        nargs=2,
+        action='append',
+        default=[],
+        help='''
+            User-facing notes for the release in the specified language,
+            e.g. "--release-notes en-US 'Bug fixes and performance improvements.'".
+            Can be specified multiple times for multiple languages.
+            For language codes see https://support.google.com/googleplay/android-developer/table/4419860
+        ''',
+        metavar=('LANGUAGE', 'TEXT')
+    )
     args = parser.parse_args()
 
     service_account_file = args.service_account_file
     package_name = args.package_name
     app_bundle_file = args.aab_file
     track = args.track
+    release_notes = list(map(lambda pair: {"language": pair[0], "text": pair[1]}, args.release_notes))
```

Comment:
I prefer using more specific names:
```suggestion
    release_notes = list(map(lambda note: {"language": note[0], "text": note[1]}, args.release_notes))
```